src/components/description-list/_description-list.scss

```diff
@@ -7,25 +7,62 @@
         clear: both;
         float: left;
         font-weight: $font-weight-bold;
-
-        &:not(:first-child) {
-            margin-top: 0.5rem;
-        }
     }
 
     &__value {
         float: right;
         margin-left: 0; /* As normalize adds a 40px left margin */
 
-        &:not(:nth-of-type(1)) {
-            @include mq(m) {
+        & + & {
+            @include mq(xs) {
+                margin-top: 0;
+            }
+        }
+    }
+
+    &__item {
+        &:not(:first-child) {
+            .ons-description-list__term {
                 margin-top: 0.5rem;
             }
+
+            .ons-description-list__value:nth-of-type(1) {
+                @include mq(m) {
+                    margin-top: 0.5rem;
+                }
+            }
+            .ons-description-list--inline & .ons-description-list__value:nth-of-type(1) {
+                @include mq(l) {
+                    margin-top: 0;
+                }
+                @include mq(xs, l) {
+                    margin-top: 0.5rem;
+                }
+            }
+            .ons-description-list--inline & .ons-description-list__term {
+                @include mq(l) {
+                    margin-top: 0;
+                }
```

Comment:
This could be reorganised to avoid the repetition of `.ons-description-list--inline` and `@include mq(l)` here

Reply:
Theres still a lot of repetition and redundant code here, all this could be replaced by just this:
```
.ons-description-list--inline & {
    .ons-description-list__value:nth-of-type(1), .ons-description-list__term {
        @include mq(l) {
            margin-top: 0;
        }
    }
}
```